src/components/MainDrawer/index.js

```diff
@@ -9,15 +9,17 @@ import PullCommit from './Tabs/PullCommit'
 import Error from './Tabs/Error'
 import PullCommitMenu from 'components/Menu/PullCommit'
 import Setting from 'components/Setting'
+import { Resizable } from 're-resizable'
 
-import { compact } from 'lodash'
+import { compact, throttle } from 'lodash'
 import * as Style from './style'
+import { useSettingContext } from 'components/Setting/Context/Provider'
 
 const useStyles = makeStyles({
   paper: {
-    width: 400,
-    display: 'flex',
-    flexDirection: 'column',
+    boxShadow:
```

Comment:
The shadow is so great! I love it.
![image](https://user-images.githubusercontent.com/20126676/130451496-340d3a25-8e00-4174-bb87-25bf0a1f98a5.png)

Reply:
I stole it <3

Reply:
I stole it <3

Reply:
I stole it <3